src/state/routing/v2Slice.ts

```diff
@@ -80,7 +80,10 @@ export const routingApiV2 = createApi({
                 // cast as any here because we do a runtime check on it being an object before indexing into .errorCode
                 const errorData = response.error.data as any
                 // NO_ROUTE should be treated as a valid response to prevent retries.
-                if (typeof errorData === 'object' && errorData?.errorCode === 'NO_ROUTE') {
+                if (
+                  typeof errorData === 'object' &&
+                  (errorData?.errorCode === 'NO_ROUTE' || errorData?.detail === 'No quotes available')
```

Comment:
This should be in a const exported by the routing-api package. If that's not available, can you make it a follow-up with the routing-api team?

Reply:
do you mean SOR? There is no routing-api package